packages/lodestar/src/api/rest/routes/beacon/version.ts

```diff
@@ -0,0 +1,9 @@
+import {IFastifyServer} from "../../index";
+import * as fastify from "fastify";
+import {IApiModules} from "../../../interface";
+
+export const registerVersionEndpoint = (fastify: IFastifyServer, modules: IApiModules): void => {
+  fastify.get<fastify.DefaultQuery, {}, unknown>('/version', {}, (request, reply) => {
+    reply.code(200).type('application/json').send("Lodestar x.x.x");
```

Comment:
Should we do like in `BeaconApi.getClientVersion()`:
```
return Buffer.from(`lodestar-${process.env.npm_package_version}`, 'utf-8');
```

Reply:
yeah probably